.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.15.3
+current_version = 1.15.4
```

Comment:
Current version is 1.15.4.  How come it's showing 1.15.3 for the diff?  Same for the templates also.

Reply:
fixed